assets/javascripts/discourse/models/chat-channel.js

```diff
@@ -0,0 +1,5 @@
+export const CHATABLE_TYPES = {
+  directMessageChannel: "DirectMessageChannel",
+  topicChannel: "Topic",
+  categoryChannel: "Category",
+};
```

Comment:
I'd expect `models/chat-channel` to export a `ChatChannel` model, not an enum 😉 It should probably live in `lib` or some such.

Reply:
Not sure about this, I agree I would expect the default to export a ChatChannel, but having the enum defined in this file does seem legit?

Reply:
If there was a default export, sure, an additional export would be fine.
But in general we want filenames/paths to fit contents. If you look at the list of models and see `chat-channel` file, what would you expect to see inside? 😃

Reply:
no sorry I still don't agree. It's not because we haven't created a chat-channel model yet that I should put this code in lib. Its properties of chat-channel model so it makes sens in this file.